crates/transaction-pool/src/traits.rs

```diff
@@ -981,7 +967,7 @@ pub trait PoolTransaction:
     type Consensus;
 
     /// Associated type representing the recovered pooled variant of the transaction.
-    type Pooled: Encodable2718 + Into<Self>;
+    type Pooled: SignedTransaction;
```

Comment:
makes sense, we don't expect non signed txs here

Reply:
yeah, it's also more like a `RecoverableTransaction`